src/main/java/PeeKay.java

```diff
@@ -0,0 +1,90 @@
+import java.util.Scanner;
+
+public class PeeKay {
+    static String indent = "     ";
+    static String line = "    ____________________________________________________________";
+    static Task[] list = new Task[100];
```

Comment:
Should plural form be used instead (lists instead of list) since this is representing a coellction of objects ?